content/en/observability_pipelines/guide/set_quotas_for_data_sent_to_a_destination.md

```diff
@@ -0,0 +1,214 @@
+
+- `vector.component_sent_event_bytes_total` (metric)
+    - `component_id:quota_example` (tag)
+    - `output:dropped` (tag)
+
+If the `event` type is specified in the configuration, then this is the combination of metric and tags that shows all events that were sent after the limit.
+
+- `vector.component_sent_events_total` (metric)
+    - `component_id:quota_example` (tag)
+    - `output:dropped` (tag)
+
+### Set up a metric monitor
+
+To set up a monitor to alert when the quota is reached:
+
+1. Navigate to the [New Monitor][5] page.
+1. Select **Metric**.
+1. Leave the detection method as **Threshold Alert**.
+1. In the **Define the metric** field:  
+    a. Enter `vector.component_sent_event_bytes_total` for the metric.  
+    b. In the **from** field, add `component_id:<transform_name>,output:dropped` and replace `<transform_name>` with the name of your `quota` transform.  
+    c. Enter `host` in the **sum by** field.  
+    d. Leave the setting to evaluate the `sum` of the query over the `last 5 minutes`.
+1. In the **Set alert conditions** section:  
+    a. Leave the setting to trigger when the evaluated value is `above` the threshold for any `host`.  
+    b. Enter `1` for the **Alert threshold**. This means that if the metric query is greater than 1, then the monitor alerts.
+See [Metric Monitors][6] for more information.
+1. In the **Notify your team** section:  
+    a. Enter a name for your monitor.  
+    b. Enter a notification message. See [Notifications][7] and [Variables][8] for more information on customizing your message.  
+    c. Select who and which services the notifications are sent to.
+1. Optionally, you can set [renotifications][9], tags, teams, and a [priority][10] for your monitor. You can also define [permissions][11] and audit notifications.
+1. Click **Create**.
+
+## Route logs sent after the limit to datadog_archives
+
+The Observability Pipelines `datadog_archives` destination formats logs into a Datadog-rehydratable format and then routes it to [Log Archives][12].
+
+The example configuration below is similar to the previous [example configuration](#handling-data-sent-after-the-limit). The only difference is that the destination type is `datadog_archives` so that all the events sent to Observability Pipelines, after the quota has been reached, are routed to the archives.
+
+```yaml
+sources:
+ generate_syslog:
+   type: demo_logs
+   format: syslog
+   interval: 1
+
+
+transforms:
+ parse_syslog:
+   type: remap
+   inputs:
+     - generate_syslog
+   source: |
+     # Parse the message as syslog
+     . = parse_syslog!(.message)
+     .environment = "demo"
+     .application = "opw"
+ quota_archiving_example:
+   type:
+     quota
+   inputs:
+     - parse_syslog
+   limit:
+     type: bytes
+     bytes:
+       max: 200000
+   window:
+     1m
+sinks:
+ archive_dropped:
+   type: datadog_archives
+   inputs:
+     - quota_archiving_example.dropped
+    bucket: "X"
+    service: "X"
```

Comment:
Need to update the `X`s

Reply:
Generalizing it with environment variable like `${DD_ARCHIVES_BUCKET}` should suffice here